docs/sql-migration-guide.md

```diff
@@ -24,6 +24,8 @@ license: |
 
 ## Upgrading from Spark SQL 3.2 to 3.3
 
+  - Since Spark 3.3, the `histogram_numeric` function in Spark SQL returns an output type of an array of structs (x, y), where the type of the 'x' field in the return value is propagated from the input values consumed in the aggregate function. In previous versions, 'x' always had double type.
```

Comment:
Can we mention which configuration should be enabled to restore the previous behaviour?

Reply:
Sure, this is done.